### ISSUE TITLE
Create unit tests

### ISSUE THREAD

**Issue by balagrivine** [tyball21/Open-HAMS#8]
I've gone through the project structure but can't find the directory for tests. It would be wise if we create unit tests for features in the application.

**Comment by ahmedivy** [comment]
Yes, testing is not implemented yet. We have plans for it in the future. 

Anyways, you are welcome to start working on it. Let me know if there are any issues in setting up the project.

**Comment by balagrivine** [comment]
I'm all good with setting up the project, I'll begin with the unit tests tomorrow. I'm glad this tech stack is exactly the same to what I was using at my previous job.

**Comment by balagrivine** [comment]
Add me as a collaborator to the project so that I can commit my changes

**Comment by ahmedivy** [comment]
You can create a PR to contribute to this project.
